src/app/campaign/store/models/flight.models.ts

```diff
@@ -1,14 +1,12 @@
 import { HalDoc } from 'ngx-prx-styleguide';
 import { filterUnderscores } from './haldoc.utils';
 import { Moment, utc } from 'moment';
+import { Creative } from './creative.models';
 
 export interface FlightZone {
   id: string;
   label?: string;
-  url?: string;
-  pingbacks?: string[];
-  fileSize?: number;
-  mimeType?: string;
+  creativeFlightZones?: { creativeId?: number | string; weight?: number; disabled?: boolean }[];
```

Comment:
Wait ... so you're not even using the `creativeFlightZones[].id`?  Why does the API even include it then?  May want to circle back on that after this merges ...

Reply:
Nope, that unique id is not used. The creatives on the flight are acted on as a whole.